moj-node/server/services/offender.js

```diff
@@ -97,12 +102,29 @@ module.exports = function createOffenderService(repository) {
     };
   }
 
+  async function getActivitiesForToday(bookingId) {
+    const activities = await repository.getActivitiesForToday(bookingId);
+
+    return activities.map(activity => {
```

Comment:
can you guarantee that `activities` will always be an array?

Reply:
because if it's not map will blow up 😉

Reply:
True :) Will add some checks.

Reply:
@romidane added a check to make sure it's OK, it's IE11 compatible 👍

Reply:
I meant as in `repository.getActivitiesForToday` makes a network call which may fail and if it fails does it return an empty array or or another data type that doesn't support the method `map`